requirements.txt

```diff
@@ -1,8 +1,7 @@
 dwave-ocean-sdk>=4.3.0
 
+notebook~=7.0.0
```

Comment:
Why limit `notebook` to `7.0.x` (minor), and not to `7.x`? Are minor releases of `notebook` expected to break something in this notebook?